tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -6898,39 +7053,17 @@ TEST_F(EnergyPlusFixture, OutputReportTabularMonthlyPredefined_FindNeededOutputV
     Real64 timeNotMet;
     std::vector<std::string> ZoneNames({"Zone1", "Zone2"});
 
-
-    for (int i=0; i < 2; ++i) {
+    for (int i = 0; i < 2; ++i) {
         SetupOutputVariable("Zone Mean Air Temperature", OutputProcessor::Unit::C, zoneTemp, "Zone", "Average", ZoneNames[i]);
 
-        SetupOutputVariable("Zone Heating Setpoint Not Met Time",
-                                OutputProcessor::Unit::hr,
-                                timeNotMet,
-                                "Zone",
-                                "Sum",
-                                ZoneNames[i]);
-        SetupOutputVariable("Zone Heating Setpoint Not Met While Occupied Time",
-                                OutputProcessor::Unit::hr,
-                                timeNotMet,
-                                "Zone",
-                                "Sum",
-                                ZoneNames[i]);
-        SetupOutputVariable("Zone Cooling Setpoint Not Met Time",
-                                OutputProcessor::Unit::hr,
-                                timeNotMet,
-                                "Zone",
-                                "Sum",
-                                ZoneNames[i]);
-        SetupOutputVariable("Zone Cooling Setpoint Not Met While Occupied Time",
-                                OutputProcessor::Unit::hr,
-                                timeNotMet,
-                                "Zone",
-                                "Sum",
-                                ZoneNames[i]);
+        SetupOutputVariable("Zone Heating Setpoint Not Met Time", OutputProcessor::Unit::hr, timeNotMet, "Zone", "Sum", ZoneNames[i]);
+        SetupOutputVariable("Zone Heating Setpoint Not Met While Occupied Time", OutputProcessor::Unit::hr, timeNotMet, "Zone", "Sum", ZoneNames[i]);
```

Comment:
@Myoldmopar Quite a few changes in this file and elsewhere in this PR are reformatting changes. Which form is correct, the new changes here or what we had before? This seems to happen with VS mostly. Either VS is interpreting the clang formatting rules incorrectly, or maybe there's some additional settings we need to declare that default differently in VS than in clangformat? Fishing for a solution - tired of wading through superfluous diffs.

Reply:
@mjwitte Seems like my own VS fault. I reinstalled clang and it revert the clang changes. Will clean this up today.

Reply:
Interesting. Reinstalled clang? I thought it was built-in for VS?

Reply:
Huh. Why not for mine... I installed it as a plugin/extension.

Reply:
clang support was added in VS2017